packages/stark-ui/src/modules/app-sidebar/components/app-sidebar.component.ts

```diff
@@ -87,25 +88,30 @@ export class StarkAppSidebarComponent implements OnDestroy, OnInit {
 
 	/**
 	 * Class constructor
+	 * @param logger - The sidebar service of the application
 	 * @param sidebarService - The sidebar service of the application
+	 * @param breakpointObserver - Utility for checking the matching state of @media queries
+	 * @param renderer - Angular Renderer wrapper for DOM manipulations.
+	 * @param elementRef - Reference to the DOM element where this directive is applied to.
 	 */
 	public constructor(
 		@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService,
 		@Inject(STARK_APP_SIDEBAR_SERVICE) public sidebarService: StarkAppSidebarService,
-		public breakpointObserver: BreakpointObserver
-	) {}
+		public breakpointObserver: BreakpointObserver,
+		protected renderer: Renderer2,
```

Comment:
Well, the `renderer` and the `elementRef` should always be injected so they can be passed to the constructor of the `AbstractUiComponent`.

And about the **public** keyboard, you're right, I'll put them as **protected** as in the `AbstractUiComponent`

Reply:
Mmm...
Latest remark still about this because I think I didn't explain what I have in mind correctly :s

I think you are declaring variables renderer and elementRef twice.
Once in your constructor, once in the AbstractStarkUiComponent.
Shouldn't it be better to write your constructor like this?
```typescript
public constructor(
		@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService,
		@Inject(STARK_APP_SIDEBAR_SERVICE) public sidebarService: StarkAppSidebarService,
		public breakpointObserver: BreakpointObserver,
		renderer: Renderer2,
		elementRef: ElementRef
	) {
		super(renderer, elementRef);
	}
```

Or will it be the same result?

Except of this, everything seems ok 👌

Reply:
Ahhh true!, my bad... I completely misunderstood what you said :p 

I'll fix it and then I'll merge the PR then 👍